satpy/etc/readers/vaisala_gld360.yaml

```diff
@@ -0,0 +1,43 @@
+reader:
+  description: Vaisala Global Lightning Dataset 360 reader
+  name: vaisala_gld360
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader ''
+  sensors: [vaisala_gld360]
+
+file_types:
+  vaisala_gld360:
+    file_reader: !!python/name:satpy.readers.vaisala_gld360.VaisalaGLD360TextFileHandler ''
+    file_patterns: ['flashes_{start_time:%Y%m%d}.txt']
+
+datasets:
+  datetime:
+    name: datetime
+    sensor: vaisala_gld360
+    resolution: 2000
+    file_type: vaisala_gld360
+
+  latitude:
+    name: latitude
+    sensor: vaisala_gld360
+    resolution: 2000
+    file_type: vaisala_gld360
+    standard_name: latitude
+    units: degree
```

Comment:
More precisely: `degrees_north`

Reply:
CF convention has degree_north, I'll change to that

Reply:
:+1: